--- qhull-2010.1/src/Make-config.sh.orig	2010-01-07 20:01:43.000000000 +0100
+++ qhull-2010.1/src/Make-config.sh	2010-10-16 18:27:10.719740205 +0200
@@ -114,7 +114,7 @@
 ### Man pages (trick to get around .man extension)
 
 %.1: %.man
-        cp $< $@
+	cp $< $@
 CLEANFILES = *.1
 man_MANS = rbox.1 qhull.1
 
@@ -183,6 +183,7 @@
 
 # from (frequently used files at end):
 libqhull_la_SOURCES = \
+  rboxlib.c \
   user.c \
   global.c \
   random.c \
@@ -195,7 +196,9 @@
   geom.c \
   poly.c \
   qset.c \
-  mem.c
+  mem.c \
+  usermem.c \
+  userprintf.c
 
 # how:
 libqhull_la_LDFLAGS = -version-info 4:0:0 -lm
@@ -233,7 +236,8 @@
   merge.h \
   libqhull.h  \
   qset.h \
-  user.h
+  user.h \
+  random.h
 
 
 ### Example programs
@@ -247,9 +251,7 @@
   user_eg.c \
   user_eg2.c \
-  qhull_interface.cpp \
-  Makefile.txt \
   Make-config.sh \
-  MBorland
+  Mborland
 
 doc_DATA = Changes.txt \
     index.htm \
